CHANGELOG.md

```diff
@@ -14,7 +14,7 @@ Main (unreleased)
 
 - The experimental dynamic configuration feature has been removed in favor of Flow mode. (@mattdurham)
 - The `oracledb` integration configuration has removed a redundant field `metrics_scrape_interval`. Use the `scrape_interval` parameter of the integration if a custom scrape interval is required. (@schmikei)
-- Upgrade the embedded windows_exporter to the last version. The windows_exporter contains some breaking changes.
+- Upgrade the embedded windows_exporter to the v0.22 version. The windows_exporter contains some breaking changes.
```

Comment:
```suggestion
- Upgrade the embedded windows_exporter to the v0.22 version. The windows_exporter contains some breaking changes:
```

Reply:
I added a disclaimer that all the black/white configs options exist for backwards compatibility and are allowed but didnt redocument them individually.